src/chem/Chain.js

```diff
@@ -36,6 +46,38 @@ class Chain {
     return this._residues;
   }
 
+  getType() {
+    return this._type;
+  }
```

Comment:
Let's avoid unnecessary getter/setters for public properties in the new code (they are all public, actually). Please remove this method and the underscore from the `.type`.

Reply:
Got it